randovania/exporter/hints/joke_hints.py

```diff
@@ -29,4 +29,6 @@
     "We now pledge sole allegiance to our liberator, the great one, Dark Samus.",
     "Defeat enemies with the Light Beam to earn Dark Ammo. Defeat enemies with the Dark Beam to earn Light Ammo.",
     "Your Friend Roster is currently empty.",
+    "It is said that reaching <Redacted> without taking any damage will unlock the Metroid Suit.",
```

Comment:
I'd skip this one. I'm jaded with people misunderstanding jokes so I don't want to risk.

Reply:
Okay then, I'll remove that one